### DISCUSSION TITLE
ImageOptimizer for GIFs

### DISCUSSION THREAD

**Discussion by wyeack** [dlemstra/Magick.NET#646] [Help]
I currently use an instance of ImageOptimizer to compress PNGs and JPEGs.  When I pass in a GIF, nothing seems to happen - how would I be able to optimize GIFs (i.e. call the ImageOptimizer on each frame and recombine).

**Comment by dlemstra** [comment]
When nothing happens this means that the ImageOptimizer could not optimize the image. Could you share an image? I might be able to take a look at it in the future and see if I can figure out a way to optimize it.
